libs/mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/sites/rest_v2_sites_158396482_gutenberg.json

```diff
@@ -1,30 +1,19 @@
 {
-  "request": {
-    "urlPattern": "/wpcom/v2/sites/158396482/gutenberg(/)?($|\\?.*)",
-    "method": "POST",
-    "headers": {
-      "Content-Type": {
-        "equalTo": "application/json; charset=UTF-8"
-      }
+    "request": {
+        "urlPattern": "/wpcom/v2/sites/158396482/gutenberg(/)?($|\\?.*)",
+        "method": "GET"
     },
-    "bodyPatterns" : [ {
-      "equalToJson" : {
-        "editor" : "aztec",
-        "platform" : "mobile"
-      }
-    } ]
-  },
-  "response": {
-    "status": 200,
-    "jsonBody": {
-      "editor_mobile": "aztec",
-      "editor_web": "gutenberg",
-      "opt_in": "{{request.requestLine.baseUrl}}/wpcom/v2/sites/158396482/gutenberg?editor=gutenberg&platform=mobile"
-    },
-    "headers": {
-      "Content-Type": "application/json",
-      "Connection": "keep-alive",
-      "Cache-Control": "no-cache, must-revalidate, max-age=0"
+    "response": {
+        "status": 200,
+        "jsonBody": {
+            "editor_mobile": "aztec",
```

Comment:
@JavonDavis We should possibly update the tests to default to using Gutenberg but making this change gets them working now anyway.